### ISSUE TITLE
[A11Y] - OMP Reader View | Side navigation headings on book landing page are not visible for some elements

### ISSUE THREAD

**Issue by Tribunal33** [pkp/pkp-lib#12660]
## Issue Description
The side navigation elements `Paperback, Ebook, Free PDF, etc...` do not have a proper `<h2>` element similar to `Published, Series` navigation elements.  When investigating I can see that these elements do have a hidden screen reader h2 tag but these should be updated to standard h2 that is visible.  On OJS It was, also, suggested changing these to `<h3>` as it is dulling thier usefulness as h2 elements.  In addition remove the h2 tag from `DOI` and `Keywords`. 

<img width="825" height="171" alt="Image" src="https://github.com/user-attachments/assets/7150cd9d-6356-49b7-a02f-e721fe7ef011" />

Here is an example of the h2 tag being used correctly for `series` side navigation

<img width="469" height="91" alt="Image" src="https://github.com/user-attachments/assets/fb6e8aac-e7d3-4d51-9965-b497dcf007f4" />

_As part of our on-going commitments to accessibility we are curating the issues that the auditors have highlighted while analyzing our systems. This would affect all users._

## How to reproduce / Current behavior
Go to OMP demo site main page /index.php/aup-demo//catalog/book/3 and search for the `<h2>` tag. it is not visible.  Should be with any book on the demo site. These are not standard in our dataset and can be added by enabling them in Settings Website > appearance > Setup > Sidebar or from plugins.  Also Publication Formats might need to be set up for some of these side elements to appear.  These are found in the individual Published books in the editorial view Publication > Published Artifacts section.  

## What application are you using?
OMP 3.5.0-4

## Stack used 
- Device: Laptop
- OS: Windows 11
- Browser: Chrome
- Version: Latest

## Testing method
Manual, using screenreader

## Action
// Describe action that can be taken to remediate the issue.
Add h2 tags instead of screen reader versions. 

## Additional information

here is one of the side components that should be fixed.
<img width="305" height="289" alt="Image" src="https://github.com/user-attachments/assets/faa6a250-aafd-439d-b608-0f2e9f00e553" />
